presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,89 @@
+  type constant. ({issue}`6300`)
+* Change representation of a `row` value. `ResultSet.getObject` now returns an instance of `io.prestosql.jdbc.Row` class, which better represents
+  the returned value. Previously a `row` value was represented as a `Map` instance, with unnamed fields being named like `field0`, `field1`, etc. 
+  You can access the previous behavior by invoking `getObject(column, Map.class)` on the `ResultSet` object. ({issue}`4588`)
+* Represent `varbinary` value using hex string representation in `ResultSet.getString`. Previously the return value was useless, similar to `"B@2de82bf8"`. ({issue}`6247`)
+* Report precision of the `time(p)`, `time(p) with time zone`,  `timestamp(p)` and `timestamp(p) with time zone` in the `DECIMAL_DIGITS` column
+  in the result set returned from `DatabaseMetaData#getColumns`. ({issue}`6307`)
+* Fix the value of the `DATA_TYPE` column for `time(p)` and `time(p) with time zone` in the result set returned from `DatabaseMetaData#getColumns`.  ({issue}`6307`)
+* Fix failure when reading a `timestamp` or `timestamp with time zone` value with seconds fraction greater than or equal to 999999999500 picoseconds. ({issue}`6147`)
+* Fix failure when reading a `time` value with seconds fraction greater than or equal to 999999999500 picoseconds. ({issue}`6204`)
+* Fix element representation in arrays returned from `ResultSet.getArray`, making it consistent with `ResultSet.getObject`.
+  Previously the elements were represented using internal client representation (e.g. `String`). ({issue}`6048`)
+* Fix `ResultSetMetaData.getColumnType` for `timestamp with time zone`. Previously the type was miscategorized as `java.sql.Types.TIMESTAMP`. ({issue}`6251`)
+* Fix `ResultSetMetaData.getColumnType` for `time with time zone`. Previously the type was miscategorized as `java.sql.Types.TIME`. ({issue}`6251`)
+* Fix failure when an instance of `SphericalGeography` geospatial type is returned in the `ResultSet`. ({issue}`6240`)
+  
+## CLI Changes
+
+* Fix rendering of `row` values with unnamed fields. Previously they were printed using fake field names like `field0`, `field1`, etc. ({issue}`4587`)
+* Fix query progress reporting. ({issue}`6119`)
+* Fix failure when an instance of `SphericalGeography` geospatial type is returned to the client. ({issue}`6238`)
+
+## Hive Connector Changes
+
+* Allow configuring S3 endpoint in security mapping. ({issue}`3869`)
+* Add support for S3 streaming uploads. Data is uploaded to S3 as it is written, rather
+  than staged to a local temporary file. This feature is disabled by default, and can be enabled
+  using the `hive.s3.streaming.enabled` configuration property. ({issue}`3712`, {issue}`6201`)
+* Decrease default refresh thread count for metastore cache. ({issue}`6156`)
+* Reduce load on metastore when background cache refresh is enabled. Partition metadata and statistics are 
```

Comment:
It seems to me we should merge this point with one above and just use `Reduce load on metastore when background cache refresh is enabled` as description.